packages/forma-36-react-components/src/components/ControlledInput/ControlledInput.tsx

```diff
@@ -75,6 +78,7 @@ export const ControlledInput = ({
         name={name}
         checked={checked}
         type={type}
+        ref={(el) => el && (el.indeterminate = indeterminate)}
```

Comment:
Personally I'd say we should move this outside of the ref callback, it's a bit illogical to find logic here.

Reply:
done 👍